src/spasht/ui/template/agent_spasht.js.twig

```diff
@@ -38,20 +42,29 @@ function StoreIntoDb(key) {
 
 $("#detailsTable").ready(function(){
     // Hide showdetails Table
+    document.getElementById("showDetailsTableHead").value = "";
     $("#detailsTable").hide();
     $(".detailsRow").hide();
 });
 
 function showDetails(key){
 
     var detailsRow = "detailsRow"+key;
+    var definitionsRow = "definitionsRow"+key;
 
     // Hide all the Table row
     $(".detailsRow").hide();
 
+    // Remove Highlight from all
+    $(".definitionsRow").css("background-color", "white");
```

Comment:
Please make use of `.simpleTable .even` class instead of redefining the CSS
https://github.com/fossology/fossology/blob/8f692a66355e5c3acbd022b251ff39b7edf366a8/src/www/ui/css/fossology.css#L130

Reply:
Investigating for affected area, Will change soon